index.html

```diff
@@ -78,6 +78,58 @@ <h3>HTML 4.0 (1997-1999)</h3>
         </ul>
       </section>
       <hr />
-    </main>
+      <section>
+        <h2>The Rise of Modern HTML</h2>
+        <h3>XHTML (2000)</h3>
+        <p>A stricter version of HTML based on <strong>XML,</strong> designed for cleaner coding. However, it was considered too rigid and never gained wide adoption.</p>
```

Comment:
The `,` should probably be outside of the `<strong>` element here:
```suggestion
        <p>A stricter version of HTML based on <strong>XML</strong>, designed for cleaner coding. However, it was considered too rigid and never gained wide adoption.</p>
```

Reply:
Mkay